pandas/core/generic.py

```diff
@@ -4862,6 +4862,8 @@ def sample(
             Default = 1 if `frac` = None.
         frac : float, optional
             Fraction of axis items to return. Cannot be used with `n`.
+            Be careful, with a small number of rows and a small fraction the
+            result can be empty.
```

Comment:
Something like `If the computed number of samples is zero, the result will be empty.` would be better I think. It feels quite obvious to me, but if you find that surprising, happy to see it clarified.